.cargo-deny.toml

```diff
@@ -150,7 +150,7 @@ exceptions = [
 # The name of the crate the clarification applies to
 #name = "ring"
 # The optional version constraint for the crate
-version = 2
+#version = 2
```

Comment:
nit: was there any CI issue with this?

Reply:
Yes, was failing CI

Reply:
Or maybe not CI, but failed when I did `make all` locally